src/chrome/content/rules/OpenX.net.xml

```diff
@@ -18,29 +18,22 @@
 -->
 <ruleset name="OpenX.net (partial)">
```

Comment:
The question is, when does "partial" need to be part of the name and when is it not required?
I could argue that since subdomains like www do not work, it is still partial.